spec/zoneddatetime.html

```diff
@@ -1078,18 +1078,40 @@ <h1>Properties of Temporal.ZonedDateTime Instances</h1>
   <emu-clause id="sec-temporal-zoneddatetime-abstract-ops">
     <h1>Abstract operations</h1>
 
-    <emu-clause id="sec-temporal-interpretisodatetimeoffset" aoid="InterpretISODateTimeOffset">
-      <h1>InterpretISODateTimeOffset ( _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_, _offsetBehaviour_, _offsetNanoseconds_, _timeZone_, _disambiguation_, _offsetOption_, _matchBehaviour_ )</h1>
-      <p>
-        The abstract operation InterpretISODateTimeOffset determines the exact time in _timeZone_ corresponding to the given calendar date and time, and the given UTC offset in nanoseconds.
-        In the case of more than one possible exact time, or no possible exact time, an answer is determined using _offsetBehaviour_ (one of ~option~, ~exact~, or ~wall~), _disambiguation_ (one of *"earlier"*, *"later"*, *"compatible"*, or *"reject"*) and _offsetOption_ (one of *"ignore"*, *"use"*, *"prefer"*, or *"reject"*).
-      </p>
-      <p>
-        As a special case when parsing ISO 8601 strings which are only required to specify time zone offsets to minutes precision, if _matchBehaviour_ is ~match minutes~, then a value for _offsetNanoseconds_ that is rounded to the nearest minute will be accepted in those cases where _offsetNanoseconds_ is compared against _timeZone_'s offset.
-        If _matchBehaviour_ is ~match exactly~, then this does not happen.
-      </p>
+    <emu-clause id="sec-temporal-interpretisodatetimeoffset" type="abstract operation">
+      <h1>
+        InterpretISODateTimeOffset (
+          _year_: an integer,
+          _month_: an integer,
+          _day_: an integer,
+          _hour_: an integer,
+          _minute_: an integer,
+          _second_: an integer,
+          _millisecond_: an integer,
+          _microsecond_: an integer,
+          _nanosecond_: an integer,
+          _offsetBehaviour_: one of ~option~, ~exact~ or ~wall~,
+          _offsetNanoseconds_: an integer,
+          _timeZone_: an Object,
+          _disambiguation_: one of *"earlier"*, *"later"*, *"compatible"*, or *"reject"*,
+          _offsetOption_: one of *"ignore"*, *"use"*, *"prefer"*, or *"reject"*,
+          _matchBehaviour_: one of ~match exactly~ or ~match minutes~,
+        )
```

Comment:
```suggestion
        ): an integer
```

Reply:
Ditto:
```
Error: spec/zoneddatetime.html:1116:30: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1118:31: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1125:38: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1127:44: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1134:45: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1135:29: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1161:32: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1162:34: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1165:30: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1166:32: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1167:32: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1168:36: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1172:38: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1173:30: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1175:22: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1176:30: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1177:30: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1181:69: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1189:32: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1193:41: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1194:36: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1195:34: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1196:38: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:69:1: ToTemporalZonedDateTime does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:80:1: ToTemporalZonedDateTime does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:81:1: ToTemporalZonedDateTime does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:585:1: InterpretISODateTimeOffset does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:706:1: ToTemporalZonedDateTime does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:741:1: ToTemporalZonedDateTime does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:803:1: InterpretISODateTimeOffset does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:817:1: ToTemporalZonedDateTime does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:1196:1: InterpretISODateTimeOffset does not return a Completion Record, but is consumed as if it does
Error: spec/abstractops.html:516:1: InterpretISODateTimeOffset does not return a Completion Record, but is consumed as if it does
```

Reply:
```suggestion
        ): a normal completion containing an integer
```

Reply:
Done :+1: